Privacy.md

```diff
@@ -0,0 +1,87 @@
+# TrebleShot Privacy Policy
+This is the developer's policy on how your personal information may be collected, used, disclosed and secured. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized and the choices you have associated with that data. If you decide to continue using this software, it means you accept all terms and conditions stated by the developer in this policy. Kindly note that this privacy policy is restricted only to TrebleShot software and the developer is not and will not be liable to be held in account for the privacy practices of any third-party tools or services that may be contained in the software. <hr>
+
+## Permission and Access
+
+TrebleShot software requires quite a lot of permissions to function effectively and efficiently. Without some of the permissions enabled, you might have some issues and difficulties while using the software.
```

Comment:
Could we also mention that the only permission required to keep the app functioning is "storage permission" and the others are just optional? I am aware you have already stated in the last sentence already, still it seems it does not sound explicit.

Reply:
Will make the correction